### PR TITLE
Implement error pages

### PR DESCRIPTION
**Pull request by cervols**
**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [ ] screenshot of rails_best_practices and other tools added
- [x] screenshot of feature is added if task has UI

### CODE REVIEW

File: app/assets/stylesheets/errors.css.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+.error-code {
+  font-family: "Montserrat", Georgia, "Times New Roman", Times, serif;
+  padding: 1.5rem;
+  h2 { font-size: 10rem; }
+  h3 { font-size: 1.51rem; }
+}
+
+.py-2 {
+  padding-bottom: 1.5rem !important;
```
Comment (f1nwe) [inline app/assets/stylesheets/errors.css.scss]: try to avoid '!important'.

File: app/controllers/errors_controller.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class ErrorsController < ApplicationController
+  layout "errors"
+
+  def not_found
+    respond_to do |format|
+      format.html { render status: 404 }
```
Comment (f1nwe) [inline app/controllers/errors_controller.rb]: It is better to explicitly declare templates that you render.

File: app/views/layouts/errors.html.slim [hunk 3]
```diff
@@ -0,0 +1,24 @@
+doctype html
+html
+  head
+    title
+      | HeroesTeamApp
```
Comment (f1nwe) [inline app/views/layouts/errors.html.slim]: you can write it in one row
title HeroesTeamApp

File: app/views/layouts/errors.html.slim [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      .content-wrapper
+        .content-body
+          section.flexbox-container
+            .col-12.d-flex.justify-content-center
+              .col-md-4.p-0
+                = yield
+                .error-footer.card-footer.bg-transparent
+                  .row
+                    p.text-muted.text-center.col-12.py-1
+                      | Copyright © 2018&nbsp;
```
Comment (igorkasyanchuk) [inline app/views/layouts/errors.html.slim]: 2018 ??
what we will do in 1 year?

File: app/views/layouts/errors.html.slim [hunk 5]
```diff
@@ -0,0 +1,23 @@
+doctype html
+html
+  head
+    title HeroesTeamApp
```
Comment (igorkasyanchuk) [inline app/views/layouts/errors.html.slim]: do we use already meta_tags gem?
I want to see different titles on error pages

**Pull request merged**